packages/lodestar/README.md

```diff
@@ -0,0 +1,60 @@
+# Lodestar
+[![](https://img.shields.io/travis/com/ChainSafe/lodestar/master.svg?label=master&logo=travis "Master Branch (Travis)")](https://travis-ci.com/ChainSafe/lodestar)
+[![Discord](https://img.shields.io/discord/593655374469660673.svg?label=Discord&logo=discord)](https://discord.gg/aMxzVcr)
+[![codecov](https://codecov.io/gh/ChainSafe/lodestar/branch/master/graph/badge.svg)](https://codecov.io/gh/ChainSafe/lodestar)
```

Comment:
I would leave codecov and maintanability in root readme as well because they refer to whole repo and not just module.

We can separate codecov once they implement:
https://docs.codecov.io/docs/flags#section-flag-badges-and-graphs